build/bootstrap.proj

```diff
@@ -7,15 +7,20 @@
         <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
     </PropertyGroup>
 
+    <!-- After changing versions here, run configure.ps1 -->
+
     <ItemGroup>
         <PackageDownload Include="ILMerge" version="[3.0.21]" />
         <PackageDownload Include="Lucene.Net" version="[3.0.3]" />
         <PackageDownload Include="MicroBuild.Core" version="[0.2.0]" />
         <PackageDownload Include="Microsoft.Build" version="[15.1.262-preview5]" />
         <PackageDownload Include="Microsoft.DotNet.Build.Tasks.Feed" version="[5.0.0-beta.20159.1]" />  <!-- for publishing to the .NET Core build asset registry (BAR) -->
         <PackageDownload Include="Microsoft.DotNet.Maestro.Tasks" version="[1.1.0-beta.20065.7]" />  <!-- for publishing to the .NET Core build asset registry (BAR) -->
-        <PackageDownload Include="Microsoft.VisualStudio.Composition" version="[15.8.98]" />
-        <PackageDownload Include="Microsoft.VisualStudio.ProjectSystem" version="[16.0.201-pre-g7d366164d0]" />
+        <PackageDownload Include="Microsoft.VisualStudio.Composition" version="[16.4.11]" />
+        <PackageDownload Include="Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime" version="[14.3.26930]" />
+        <PackageDownload Include="Microsoft.VisualStudio.ProjectSystem" version="[16.7.156-pre]" />
+        <PackageDownload Include="Microsoft.VisualStudio.ProjectSystem.Managed" version="[16.7.0-beta1-65318-02]" />
+        <PackageDownload Include="Microsoft.VisualStudio.ProjectSystem.Managed.VS" version="[16.7.0-beta1-65318-02]" />
```

Comment:
Not sure here but is it a good idea for NuGet to depend on prerelease packages?

Reply:
If it works like the Project System VSIX, then none of these references are actually used when packaging for VS. They're only used for compilation and local debugging.

Without using these prerelease packages, we won't be able to compile against the new as-yet-unreleased APIs.